app/views/devices_guidance/how_to_order_for_independent_special_schools.html.erb

```diff
@@ -0,0 +1,215 @@
+    </p>
+
+    <h4 class="govuk-heading-s">
+      Tell us if your allocation is wrong
+    </h4>
+    <p class="govuk-body">
+      If your estimated allocation is wrong, <%= govuk_link_to 'contact us', support_ticket_path %> and let us know
+      how many laptops you currently need for eligible pupils. Do not include any personal data, such as pupils’ names or dates of birth.
+    </p>
+
+    <h3 class="govuk-heading-m">
+      2. Check what pupils need
+    </h3>
+    <p class="govuk-body">
+      We recommend contacting pupils, families or schools as soon as possible to find out the following information.
+    </p>
+
+    <h4 class="govuk-heading-s">
+      Laptops
+    </h4>
+    <p class="govuk-body">
+      To order laptops, you need to find out which:
+    </p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>
+        eligible pupils do not have access to a laptop or tablet
+      </li>
+      <li>
+        <%= govuk_link_to "Type of laptop and IT settings are most suitable for each care leaver", '/devices/device-specification' %>
+        - if they need Google Chromebooks, you should ask for their educational setting’s domain for
+        <span class="app-no-wrap">Google Workspace for Education Fundamentals</span>
+        <span class="app-no-wrap">(formerly G Suite for Education)</span>, and recovery email address. We’ll ask you for
+        these after you place your order.
+      </li>
+    </ul>
+    <p class="govuk-body">
+      <%= govuk_link_to 'Find out about safeguarding responsibilities, laptop ownership and providing IT support', '#step-6' %>.
+    </p>
+
+    <h4 class="govuk-heading-s">
+      Internet access
+    </h4>
+    <p class="govuk-body">
+      To request internet access, you need to find out which:
+    </p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>
+        eligible pupils do not have internet access at home
+      </li>
+      <li>
+        <%= govuk_link_to 'type of internet access is most suitable for each pupil', connectivity_home_path %>
+        (you also need to gather mobile phone information if you want to request extra data for pupils on participating networks)
+      </li>
+    </ul>
+    <p class="govuk-body">
+      You do not need to complete this step before moving on to step 3.
+    </p>
+
+    <h3 class="govuk-heading-m">
+      3. Complete and submit your contact details form
+    </h3>
+    <p class="govuk-body">
+      We need some information from you to set up your Get help with technology service account, even if you’ve used
+      the service before. You’ll place your orders using this account.
+    </p>
+    <p class="govuk-body">
+      If you’ve been nominated to order for your LA, you’ll receive an email with a link to an online form.
+      You need to use that form to tell us:
+    </p>
+    <ol class="govuk-list govuk-list--number">
+      <li>
+        Your full name.
+      </li>
+      <li>
+        Your local authority name and 3-digit code.
```

Comment:
is there somewhere the user could go to find out what their 3 digit code might be? A bit like the Get schools information service which schools can use to get their URN

Reply:
No, most users refer to previous orders or correspondence.